software-containers/README.md

```diff
@@ -18,65 +18,13 @@ group:      sss files
 `xterm` must be installed on the container
 ## Creating a CASA container
```

Comment:
You can change this header to be "Building a CASA container" and then only have the steps.

Reply:
Done.